aptos-move/framework/src/natives/aggregator_natives/aggregator_v2.rs

```diff
@@ -270,6 +282,42 @@ fn native_try_sub(
     Ok(smallvec![Value::bool(success)])
 }
 
+fn native_is_at_least_impl(
```

Comment:
does this (and can it, if not) have a simple unit test?

Reply:
there is unit tests on the delayed_field_extension.rs, and there are move e2e tests for native calls.